provider/aws/formation/rack.json

```diff
@@ -3228,23 +3252,92 @@
         }]
       }
     },
+    "CMKPolicy": {
+      "Type": "AWS::IAM::ManagedPolicy",
+      "Properties": {
+        "Description": "Access to the Customer Managed key for iam roles",
+        "Path": "/convox/",
+        "PolicyDocument": {
+          "Version": "2012-10-17",
+          "Statement": [
+            {
+              "Effect": "Allow",
+              "Action": [
+                "kms:Encrypt",
+                "kms:Decrypt",
+                "kms:ReEncrypt*",
+                "kms:GenerateDataKey*",
+                "kms:DescribeKey"
+              ],
+              "Resource": "*"
```

Comment:
We should probably reference `CustomerManagedKey` here so we only allow those actions for the key we're creating. There might be other KMS keys in the AWS account that we don't care about.